emloop/api.py

```diff
@@ -206,7 +211,16 @@ def create_hooks(config: dict, model: Optional[AbstractModel]=None, dataset: Opt
     return hooks
 
 
-def create_main_loop(config: dict, output_root: str, restore_from: str=None) -> MainLoop:
+def clean_output_dir(output_dir: str, delete_dir: bool) -> None:
```

Comment:
It does not feel right to pass a bool flag just to either do something or do nothing, Yes you saved like 5 lines elsewhere but we really should call this function only when needed.

By the way, the name is a little bit off too. I would assume that the dir would still exist after the *cleaning*, lets just rename it to `delete_output_dir`. Which is still not the best option as it would simply log and delete any dir, not just the output one so `delete_dir` should do it :).